BlogTemplate/Pages/Post.cshtml.cs

```diff
@@ -24,14 +24,13 @@ public PostModel(BlogDataStore dataStore, MarkdownRenderer markdownRenderer)
         public CommentViewModel NewComment { get; set; }
 
         public Post Post { get; set; }
-
         public HtmlString HtmlBody()
         {
             var html = _markdownRenderer.RenderMarkdown(Post.Body);
             return html;
         }
 

```

Comment:
just make sure to be careful with any merge conflicts with master on post.cshtml.cs page, I have had some issues with it in the past.